src/components/Templates/DefaultSlideTemplate.vue

```diff
@@ -18,77 +15,64 @@
         lg6>
         <v-card
           color="transparent"
-          flat
-          height="100px">
+          flat>
           <v-card-title
             primary
             class="title">
-            <h2 class="display-3">{{ title }}</h2>
+            <h2 class="display-3">{{ slideView.title.content }}</h2>
           </v-card-title>
           <v-card-title
             primary
             class="title">
-            <h2 class="display-2">{{ time }}</h2>
+            <h2 class="display-2">{{ slideView.date.content }}, {{ slideView.time.content }} </h2>
           </v-card-title>
         </v-card>
       </v-flex>
       <!-- Done Event title card -->
 
       <!-- Carousel Component -->
       <v-flex
-        d-flex
         xs12
-        sm8
-        md6>
+        sm12
+        md12
+        lg6>
         <v-carousel
+          :value="carousel"
           hide-controls
           hide-delimiters
-          interval="40000/6">
+          :interval="carouselInterval">
           <v-carousel-item
-            v-for="(item,i) in items"
+            v-for="(item,index) in slideView.items"
             :src="item.src"
-            :key="i"/>
+            :key="index"/>
         </v-carousel>
       </v-flex>
       <!-- Done the Carousel item -->
 
       <v-flex
-        d-flex
-        xs12
-        v-show="!slideModeOn">
+      xs12>
         <v-card
           color="transparent"
           flat
-          height="40px"/>
+          height="45px"/>
       </v-flex>
     </v-layout>
 
     <!-- Component used as Event Body -->
-    <v-layout
-      row
-      wrap
-      fluid>
-      <v-flex
-        xs12
-        v-show="!slideModeOn">
-
-        <v-card
-          color="white"
-          height="300px">
-          <v-container
-            fill-height
-            fluid>
-            <v-layout align-center>
-              <v-flex>
-                <div class="title mb-3"> {{ subtitle }}</div>
-                <v-divider class="my-3"/>
-                <span class="title mb-3"> {{ eventBody }} </span>
-              </v-flex>
-            </v-layout>
-          </v-container>
-        </v-card>
-      </v-flex>
-    </v-layout>
+    <template>
+      <v-jumbotron color="grey lighten-3">
+        <v-container fill-height>
+          <v-layout align-center>
+            <v-flex>
+              <v-divider class="my-3"/>
+              <h3 class="headline"> {{ slideView.eventBody.content }}</h3>
```

Comment:
Is _eventBody_.content here the same thing as the Slide module index's currentSlide._description_.content? (as well as the other font fields to I suppose)
`currentSlide: {
+    // the title of the slide, what it is called by humans, has text, font information and color.
+    title: { content: '', fontColor: null, fontSize: null, fontStyle: null, fontWeight: null },
+    // similar to title, but generally larger text content.
+    **description: { content: '', fontColor: null, fontSize: null, fontStyle: null, fontWeight: null },**
+    // image array of image files (or link to image files?) to be displayed in the slide
+    images: [],
+    // the date of the slides event, has similar font info as title/description, content is date object
+    date: { content: null, fontColor: null, fontSize: null, fontStyle: null, fontWeight: null },
+    // the time of the event, same as date but time object instead of date object
+    time: { content: null, fontColor: null, fontSize: null, fontStyle: null, fontWeight: null },
+
+    meta: { template: null, timeout: '', repeatable: false, startDate: '', endDate: '' }`

In index.js file from:
https://github.com/371team4/371Team4/pull/80/files

If they are the same should we change them to both use the same terminology?
@KingMackle

Reply:
Good question, to be honest I am not quite sure. I would also like to clarify with @KingMackle

Reply:
@Masonfrith Yes we should have the same terminology for everything, thanks for catching that one. @a-kkwan Can you please change line 88 to `slide` and line 102 to `description`, also this will mean changing the lines that Mason mentioned as well. Thanks

Reply:
I have changed and pushed them accordingly, @Masonfrith @KingMackle